noetic/distribution.yaml

```diff
@@ -6816,6 +6816,19 @@ repositories:
       url: https://github.com/autoware-ai/qpoases_vendor.git
       version: master
     status: maintained
+  qt_advanced_docking_system:
+    release:
+      packages:
+      - qt_advanced_docking
+      tags:
+        release: release/noetic/{package}/{version}
+      url: https://github.com/tesseract-robotics-release/qt_advanced_docking_system-release.git
+      version: 3.8.2-3
+    source:
+      type: git
+      url: https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System.git
+      version: 3.8.2
```

Comment:
@Levi-Armstrong `version` should be a branch. That's why the CI check is failing.

Reply:
The documentation when going through the release says it should normally be MAJOR.MINOR.PATCH see below.

![image](https://user-images.githubusercontent.com/9803128/161565998-bbd7484c-72f0-4214-934d-d78e4571b44c.png)

Reply:
Okay. I have fixed both requests.